app/controllers/rockles_controller.rb

```diff
@@ -21,7 +31,7 @@ def create
 
   def show
     @rockle = Rockle.find(params[:id]) #TODO: check params
-    render 'show'
+    @rockle.user = User.find(@rockle.user_id)
```

Comment:
Active Record already knows how to find @rockle.user because of the association we declared in the rockle.rb. So you  don't need this line.